src/index.html

```diff
@@ -1,13 +1,496 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
+    <title>WITCH OF MOUNTAIN WINDS</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link 
+      rel="shortcut icon" 
+      href="./images/svg/logo-colour.svg" 
+      type="image/png"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Hello Mate Academy</h1>
-    <script type="text/javascript" src="scripts/main.js"></script>
+  <body class="page__body">
+    <header class="header page__section">
+      <div class="container header__content">
+        <div class="header__top">
+          <div class="header__menu">
+            <a href="#menu" class="icon icon__menu header__nav--link"></a>
+          </div>
+          <div class="header__logo">
+            <a href="#">
+              <div class="icon icon__logo"></div>
+            </a>
+          </div>
+          <ul class="header__nav">
+            <li>
+              <a href="#shop" class="header__nav--link">
+                shop
+              </a>
+            </li>
+            <li>
+              <a href="#contacts" class="header__nav--link">
```

Comment:
According to BEM methodology, you should use double dashes for modifiers. Change 'header__nav--link' to 'header__nav-link'.